server/src/main/java/org/opensearch/action/search/ListPitInfo.java

```diff
@@ -60,4 +54,31 @@ public void writeTo(StreamOutput out) throws IOException {
         out.writeLong(creationTime);
         out.writeLong(keepAlive);
     }
+
+    static final ConstructingObjectParser<ListPitInfo, Void> PARSER = new ConstructingObjectParser<>(
+        "list_pit_info",
+        true,
+        args -> new ListPitInfo((String) args[0], (long) args[1], (long) args[2])
+    );
+
+    static {
+        PARSER.declareString(constructorArg(), new ParseField("pit_id"));
```

Comment:
Please use CREATION_TIME, PIT_ID and KEEP_ALIVE here (this is one of the reasons to have the static:

```
    static {
        PARSER.declareString(constructorArg(), PIT_ID);
        PARSER.declareLong(constructorArg(), CREATION_TIME);
        PARSER.declareLong(constructorArg(), KEEP_ALIVE);
    }

``